src/sentry/seer/similarity/similar_issues.py

```diff
@@ -200,19 +201,19 @@ def get_similarity_data_from_seer(
                 },
             )
         except SimilarGroupNotFoundError:
-            # This will similarly mark the entire request as errored even if it's only one group
-            # that we can't find, but again, we're okay with that because a group being missing
-            # implies there's something wrong with our deletion logic, which is a problem to which
-            # we want to pay attention.
-            #
-            # TODO: The deletion logic (tell Seer to delete hashes as soon as they're deleted on the
-            # Sentry side) comes with an inherent slight delay - the request to Seer happens async, and
-            # requests between services aren't instantaneous. It's therefore possible for us to land
-            # in a race condition, where Seer recommends a hash it doesn't know it's about to be
-            # asked to delete, and we come up empty because on the Sentry side the hash has already
-            # been deleted.
             parent_hash = raw_similar_issue_data.get("parent_hash")
 
+            # Tell Seer to delete the hash from its database, so it doesn't keep suggesting a group
+            # which doesn't exist
+            delete_seer_grouping_records_by_hash.apply_async(args=[project_id, [parent_hash]])
```

Comment:
Could you add the feature check for `projects:similarity-embeddings-delete-by-hash` before making the call here? The reason I didn't have it in the function is because it calls itself and I didn't want it to be doing checks every time

Reply:
UPDATE: Had a convo IRL. The project itself isn't in scope in this function, and while we have the id, if we can avoid an extra DB call I'd like to, given this runs during ingest. So we agreed that for now I'm not going to add the flag check but also will wait to merge this until we've given the deletion a chance to run on our internal projects for a few days, to make sure it feels safe to make the deletion call without checking the flag.